pkg/controller/master-controller-manager/rbac/mapper.go

```diff
@@ -46,8 +46,8 @@ const (
 )
 
 const (
-	saSecretsNamespaceName = "kubermatic"
-
+	saSecretsNamespaceName              = "kubermatic"
+	kubeOneNamespacePrefix              = "kubeone"
```

Comment:
Is the prefix "kubeone" or actually "kubeone-"? Is this a constant somewhere? Should it be?

Reply:
the prefix is "kubeone-" but using "kubeone" as "-" is just a separator.
Added the constant here: https://github.com/kubermatic/kubermatic/pull/9479/files#diff-e516bce9ed18c3714772792cec3dfd381f170c061c2de1424d0c966e23a0d712R548

Reply:
added the constant `KubeOneNamespacePrefix` https://github.com/kubermatic/kubermatic/pull/9479/files#diff-e516bce9ed18c3714772792cec3dfd381f170c061c2de1424d0c966e23a0d712R548 here, but can not import it in rbac/mapper